assignment2/config.js

```diff
@@ -0,0 +1,43 @@
+
+var express = require("express");
+var mongoose = require("mongoose");
+var passwordHash = require("password-hash");
+var app = express();
+var conn = mongoose.createConnection(
+    "mongodb://localhost:27017/newuser",
+    {
+        useNewUrlParser: true,
+        useFindAndModify: true,
+        useCreateIndex: true,
+        useUnifiedTopology: true,
+    },
+    function (err, db) {
+        if (err) {
+            console.log("no");
+        }
+    }
+);
+var usersprofile_schema = mongoose.Schema(
+    {
+        username: {
+            type: String,
+            required: true,
+        },
+        password: {
+            type: String,
+            required: true,
+        },
+        email: {
+            type: String,
+            required: true,
+        },
+    },
+    {
+        strict: true,
+        collection: "newuserdata",
+    }
+);
+var user = conn.model("newuserdata", usersprofile_schema);
+
+exports.conn = conn;
+exports.user = user;
```

Comment:
move this schema to a separate file
under a folder `models`

Reply:
moved